lib/matplotlib/backends/backend_ps.py

```diff
@@ -892,7 +892,7 @@ def _print_figure(
         metadata must be a dictionary. Currently, only the value for
         the key 'Creator' is used.
         """
-        isEPSF = format == 'eps'
+        is_epsf = format == 'eps'
```

Comment:
Whatever the 'f' stands for...

Reply:
looks like just an alternate extension for eps... https://en.wikipedia.org/wiki/Encapsulated_PostScript

Reply:
Would change to is_eps then. It‘s equal, more common, and the string we use for the format.

Reply:
done